Clarify "Linux dependencies" for vSphere (#960)
rancher/rancher-docs: docs/how-to-guides/new-user-guides/launch-kubernetes-with-rancher/use-new-nodes-in-an-infra-provider/vsphere/create-a-vm-template.md

## Linux Dependencies

> The packages that need to be installed on the template are listed below. These will have slightly different names based on distribution; for example, some distributions ship these dependencies by default. The dependencies listed here are required for the functioning of the Rancher cluster provisioner. The cluster provisioner automatically installs additional dependencies required for Kubernetes:

Author: > The cluster provisioner automatically installs additional dependencies required for Kubernetes:

Ending that sentence with a colon makes it look like the listed dependencies are the ones required for Kubernetes. The whole point of this PR is to clarify that they are not. I'll fix this!

## Linux Dependencies

The packages that need to be installed on the template are listed below. These will have slightly different names based on distribution; some distributions ship these by default, for example.

> These dependencies are souly what is required for Rancher's cluster provisioner to work.

Reviewer: ```suggestion
These dependencies are required for the Rancher cluster provisioner.
```

> Additional dependencies required by Kubernetes will be installed automatically by the cluster provisioner.

Reviewer: ```suggestion
The cluster provisioner automatically installs additional dependencies required by Kubernetes.
```
